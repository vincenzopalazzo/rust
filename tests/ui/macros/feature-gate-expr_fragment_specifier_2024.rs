//@  compile-flags: --edition=2024 -Z unstable-options

macro_rules! m {
    ($e:expr_2021) => { //~ ERROR: expr_2021 is experimental
        $e
    };
}

fn main() {
    m!(());
}
