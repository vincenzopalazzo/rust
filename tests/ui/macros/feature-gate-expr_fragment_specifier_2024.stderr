error: expr_2021 is experimental
  --> $DIR/feature-gate-expr_fragment_specifier_2024.rs:4:6
   |
LL |     ($e:expr_2021) => {
   |      ^^^^^^^^^^^^

error: aborting due to 1 previous error

